pyccl/core.py

```diff
@@ -179,6 +180,9 @@ class Cosmology(CCLObject):
         extra_parameters (:obj:`dict`, optional): Dictionary holding extra
             parameters. Currently supports extra parameters for CAMB, with
             details described below. Defaults to None.
+        T_ncdm (:obj:`float`): Non-CDM temperature in units of photon
+            temperature. The default is ``pyccl.physical_constants.T_ncdm``.
+            # TODO: Move after T_CMB for CCLv3.
```

Comment:
Don't include # TODO in docstring

Reply:
oh yes, what was I thinking! Thanks, good catch!